src/codegen/mod.rs

```diff
@@ -214,7 +221,10 @@ impl<'a> CodegenResult<'a> {
         self.vars_seen.insert(name.into());
     }
 
-    fn inner<F>(&mut self, cb: F) -> Vec<proc_macro2::TokenStream>
+    fn inner<F>(
```

Comment:
This function is only changed to support the c++ namespace thing I did... which I don't think works.